lib/tasks/db.rake

```diff
@@ -25,4 +25,11 @@ namespace :db do
       exec "pg_restore --verbose --clean --no-acl --no-owner -h #{config[:host]} -d #{config[:database]} #{path}"
     end
   end
+
+  desc "Reset postgres pk sequence id's on all tables"
+  task seq_ids_reset: [:environment] do
+    ActiveRecord::Base.connection.tables.each do |t|
+      ActiveRecord::Base.connection.reset_pk_sequence!(t)
+    end
+  end
```

Comment:
Does this work without error on `impressions`? Curious because impressions has a uuid pk.

Reply:
Good catch, I haven't checked. Adding this to my list of things to review as well.

Reply:
From a dry-run of this Rake task and some reading. It doesn't look like this will affect Impressions since they are using UUID as pk and have no sequence. (Eg. `Impression.first` or `Impression.last` will not always be the same.)